apps/openmw/mwmechanics/levelledlist.hpp

```diff
@@ -80,7 +82,50 @@ namespace MWMechanics
                 return getLevelledItem(ref.getPtr().get<ESM::CreatureLevList>()->mBase, true, failChance);
         }
     }
+}
 
+//Returns list of items in group. You get them by returned value->mId.
+inline const std::vector<ESM::LevelledListBase::LevelItem>& itemsInGroup(const std::string& group)
+{
+    MWWorld::ManualRef ref (MWBase::Environment::get().getWorld()->getStore(), group);
+    const ESM::ItemLevList* levItem = ref.getPtr().get<ESM::ItemLevList>()->mBase;
```

Comment:
You can get to the ESM::ItemLevList without constructing an item based on it.
```
const ESM::ItemLevList* levItem = MWBase::Environment::get().getWorld()->getStore().get<ESM::ItemLevList>().find(group);
```

And that makes the itemsInGroup function so short that I would remove it all together, just place the code inline.